notebooks/official/feature_store/sdk_feature_store.ipynb

```diff
@@ -0,0 +1,2418 @@
+{
```

Comment:
<p>this section is unnecessary since we already have list_entity_types and list_features in the previous section, can you utilize the search created features in the blow section and list all features in a specific feature store?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/284/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
Ya, realized that the first search is listing out all the features.